src/core/services/interaction/interaction.spec.js

```diff
@@ -0,0 +1,26 @@
+describe("$mdInteraction", function() {
```

Comment:
I'm already working on more tests, should I provide tests for the specific components or only for the interaction service (for example mouse -> I already tried to add a mouse test, but phantomjs doesn't allows that) -> But I will look for it.